distreqx/bijectors/triangular_linear.py

```diff
@@ -0,0 +1,105 @@
+"""Triangular linear bijector."""
+
+from typing import Tuple
```

Comment:
I think I might do this as well, but for >3.8 we can actually just use `tuple` rather than `Tuple`